src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -84,7 +84,8 @@ public static void OnStopEventWritten(string name, object payload)
         tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeUrlScheme, context.Request.Scheme));
         tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeHttpResponseStatusCode, TelemetryHelper.GetBoxedStatusCode(context.Response.StatusCode)));
 
-        var httpMethod = RequestMethodHelper.GetNormalizedHttpMethod(context.Request.Method);
+        var requestMethodHelper = new RequestMethodHelper(string.Empty);
```

Comment:
Would you please promote this to a field on the class so we don't need to `new` an instance on each request?